### ISSUE TITLE
Hub definition

### ISSUE THREAD

**Issue by edunnwe1** [openconnectome/Statistical-Connectomics-Class#222]
It occurred to me that if we define the hub the way we did in class, then the degree will include the extent to which a node has self loops: a node with a lot of self loops will have a higher degree than one that may share more edges with other nodes but has no self loops. So maybe we'd want to consider this in the definition...

Also, I wonder if for questions like disease we would want a different definition for hubs for a different reason. I think the idea behind looking at hubs might be that were we to have damage in that particular region, many paths in the brain would be disrupted. So perhaps for studying the diseased brain via graphs we want to define a special case of hubs or define another term that describes nodes that disrupt the only path between a pair of other nodes? Or is there a term for that already?

Not sure I stated that quite clearly let me know if I should clarify.

**Comment by edunnwe1** [comment]
Ah, maybe I posted too soon: I think that the term I'm looking for is betweenness centrality, or "the fraction of shortest paths in the network that pass through this vertex" from http://home.kpn.nl/stam7883/graph_introduction.html. 
Can't seem to find the paper that Inigo, Sandra and Sandya presented on in the articles folder and not sure of the exact title, did that paper use something like this? Seems like it might be useful for brain diseases in particular.

**Comment by SandyaS72** [comment]
Yes the paper did measure betweenness centrality as a general characteristic of the network, but it wasn't their metric for performance or resilience to attack.
As far as the paper we presented, here is the link: http://brain.oxfordjournals.org/content/early/2014/06/18/brain.awu132
